### PR TITLE
Fixes portals moving anchored atoms

### PR DESCRIPTION
**Pull request by imtakingabreakdontatme**
:tada:

@kilkun

### CODE REVIEW

**Comment by Kromgar** [comment]
#notafixitsarebalance

**Comment by KorPhaeron** [comment]
Anchored objects aren't supposed to be moved by outside forces, this is consistent with the rest of the codebase

**Comment by Kromgar** [comment]
Guess ill just have to use jaunters to flee the megafauna after getting it on the shuttle now...

File: code/modules/mining/equipment.dm [hunk 1]
```diff
@@ -145,7 +145,7 @@
 	desc = "A stable hole in the universe made by a wormhole jaunter. Turbulent doesn't even begin to describe how rough passage through one of these is, but at least it will always get you somewhere near a beacon."
 
 /obj/effect/portal/wormhole/jaunt_tunnel/teleport(atom/movable/M)
-	if(istype(M, /obj/effect))
```
Comment (Supermichael777) [inline code/modules/mining/equipment.dm]: isnt this checking if the portal is anchored

Comment (KorPhaeron) [inline code/modules/mining/equipment.dm]: No

**Comment by Kromgar** [comment]
Wait does this mean durands cant knockback drakes by punching them now?

**Comment by kilkun** [comment]
:tada: :raised_hands: :tada:

**Comment by kilkun** [comment]
one flaw: bullets are anchored.

**Comment by ChangelingRain** [comment]
incorrect!

**Comment by kilkun** [comment]
oh did pixel projectiles change that?

**Comment by XDTM** [comment]
Mechs are currenly non-teleportable as far as i know

**Comment by Kromgar** [comment]
I used to use the hand tele to teleport my mechas... you cant even use the teleporters now?

**Comment by XDTM** [comment]
hell, the mech wormhole generator is pointless because of this

also telepads haven't worked for months on mechs either

**Comment by XDTM** [comment]
also if i'm reading the diff right this means that mechs will now be able to teleport

**Comment by AnturK** [comment]
Anchored has meaning overload.

**Pull request merged**